content/fa17/ann/azur_garris/index.md

```diff
@@ -12,8 +12,22 @@ topic: "Stucco"
 intro: ""
 
 pages:
-  - title: ""
-    link: ""
-    desc: ""
-
+- title: "Pages"
+  link: "pages.md"
```

Comment:
We should not add these pages.md files. They are not made by the students. I added them in to help with filling out the index.md files.

Reply:
Yes, you're right, Greg. Because this is a larger problem across multiple files, I am going to have a dedicated campaign to check for and remove these by another student #50 and #49